config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
I see where you're coming from, but I think `interfaces` will only suggest the use of Go interfaces. Maybe something like `Standard library`? Or perhaps this is something that should be in the `README` or test suite?